app/src/main/java/ar/com/wolox/android/example/ui/training/login/LoginFragment.java

```diff
@@ -1,26 +1,73 @@
 package ar.com.wolox.android.example.ui.training.login;
 
+import android.content.SharedPreferences;
+import android.os.Build;
+import android.support.annotation.RequiresApi;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
 import ar.com.wolox.android.R;
 import ar.com.wolox.wolmo.core.fragment.WolmoFragment;
+import butterknife.BindView;
 
 /**
  * LoginFragment is used for user to Login
  */
-public class LoginFragment extends WolmoFragment {
+public class LoginFragment extends WolmoFragment<LoginPresenter> implements ILoginView {
+    @BindView(R.id.edittext_email) EditText editTextEmail;
+    @BindView(R.id.edittext_password) EditText editTextPassword;
+    @BindView(R.id.login_button) Button loginButton;
```

Comment:
Every variable that lives inside a class we write it like: 'm + (function) + (view)'. For example: mLoginButton